components/profile-controller/config/overlays/kubeflow/namespace-labels.yaml

```diff
@@ -0,0 +1,3 @@
+# Below is a list of labels to be enforced.
+istio-injection: ''
```

Comment:
Shall we comment them out? This is not default value that work for others.

Reply:
Yes I will comment the enforcement here. I will add those labels from gcp-blueprint side.